src/Raven.Studio/typescript/viewmodels/shell.ts

```diff
@@ -148,7 +150,7 @@ class shell extends viewModelBase {
             (settings, db) => new saveStudioConfigurationCommand(settings, db).execute()
         );
         
-        this.detectBrowser();
+        this.browserAlert = new detectBrowser(true);
```

Comment:
join declaration and assignment (just like in welcome.ts file)

Reply:
Can't - has to be after `studioSettings.default.configureLoaders`
otherwise I get => `Global settings loader not found`

Unless we move the `configureLoaders `also outside of constructor...

Reply:
ok so let's leave it